### ISSUE TITLE
Works well locally but 500 error after deployed

### ISSUE THREAD

**Issue by inyourcare** [firebase/firebase-tools#6926]
### [REQUIRED] Environment info

**firebase-tools:**13.5.2

**Platform:**Windows

### [REQUIRED] Test case

here you can see my code 'https://github.com/inyourcare/fun-app'
when I try to enter the page with path 'port/board/list/0' (dynamic routing from NextJS)
500 server error comes.
It works perfectly on local and emulator.

### [REQUIRED] Steps to reproduce

contents in .env
```
NEXT_PUBLIC_KAKAO_REST_API_KEY=
NEXT_PUBLIC_KAKAO_JAVASCRIPT_KEY=
NEXT_PUBLIC_NAVER_CLIENT_ID=

GOOGLE_CLIENT_ID=
GOOGLE_CLIENT_SECRET=

EMAIL_SERVER_HOST=
EMAIL_SERVER_PORT=
EMAIL_SERVER_USER=
EMAIL_SERVER_PASSWORD=
EMAIL_FROM=
DATABASE_URL="file:./dev.db"
```
yarn
npx prisma migrate dev --name init
npx prisma generate
yarn run dev

### [REQUIRED] Expected behavior

It should work when deployed as well.

### [REQUIRED] Actual behavior

Only I can see 500 error through Chrome's console.

**Comment by leoortizz** [comment]
Hey @inyourcare, I was able to reproduce your issue and saw this error:
`Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.`

You can replicate it locally running the emulators with `NODE_ENV=production firebase emulators:start`.

The issue seem to be with your project, can you please check? 

This discussion might help: https://github.com/prisma/prisma/discussions/19669

**Comment by inyourcare** [comment]
You right. I thought the simulator automatically had the production env but it was wrong.
I tried the below and It works

generator client {
  provider = "prisma-client-js"
  output   = "./generated/client"
}
import { PrismaClient } from "@/prisma/generated/client";

But It seems pages using dynamic routing are deployed on Functions and don't have the volume db.
The database url setup is

DATABASE_URL="file:./dev.db"

And It works fine at a page in '/fun/voice_recognition/voca_US' (use prisma client, not dynamic routing)
I also tried to make client-side page calling api to bring data and the api also had the same problem (can not access the file db).
I think this might be a limitation of such database like sqlite.
Hopefully would there be some recommendations? 
Should I use database on Google Cloud?
